### ISSUE TITLE
saran

### ISSUE THREAD

**Issue by tanwirulqulub999** [pirmansx/mbf#8]
Saran dari saya untuk di bagian warna di tambahin lagi
dari kodingan MBF dulu yang di bagian def tampil(x): di tambahin warna 30 untuk gray dan sebagainya

**Comment by hantu-laut** [comment]
Dear MicroClone.

Tolong pull request aja disini. Ga usah kasih saran segala oke.. di github lu bisa kasih saran dengan hasil kode yang lu berikan

`abu_abu = "\033[1;30m \033[0m"`

**Comment by tanwirulqulub999** [comment]
yeah saya tau itu mamank, buat kode ngewarnain abu-abu / gray / grey itu \033[30;1m emang kenapa yah? biar grafiknya semakin keren

**Comment by akhunkabir** [comment]
Kena checkpoint terus

**Comment by tanwirulqulub999** [comment]
wkwkwk sorry gan, gw sebenarnya masih awam soal github jadinya gini wkwkwk seharusnya di pulls request kan?
